crates/turborepo-ffi/src/lib.rs

```diff
@@ -43,3 +45,46 @@ pub extern "C" fn get_turbo_data_dir() -> Buffer {
     let dir = dirs.data_dir().to_string_lossy().to_string();
     proto::TurboDataDirResp { dir }.into()
 }
+
+#[no_mangle]
+pub extern "C" fn changed_files(buffer: Buffer) -> Buffer {
+    let req: proto::ChangedFilesReq = buffer.into_proto().expect("buffer is valid protobuf");
```

Comment:
I think returning an error to Go instead of panicking here would be better. Go prints a ton of scary stuff if we panic here